### PR TITLE
* Updated lib with new analyze queries parameter on identity limit en…

### PR DESCRIPTION
**Pull request by paulmozo**
…d point

* Updated the identity example with new functionality
* Updated identity test to test for new functionality

### CODE REVIEW

File: lib/DataSift/Account/Identity/Limit.php [hunk 1]
```diff
@@ -82,15 +89,21 @@ public function create($identity, $service, $total_allowance)
      * 
      * @param string    $identity
      * @param integer   $limit
```
Comment (nathanmac) [inline lib/DataSift/Account/Identity/Limit.php]: This doc item should be fixed ($service)

**Comment by nathanmac** [comment]
Other than that LGTM

**Comment by paulmozo** [comment]
Cool, that's fixed now :)

**Comment by paulmozo** [comment]
@nathanmac Ok to merge + release now?

**Pull request merged**